packages/node_modules/@webex/plugin-meetings/src/meetings/util.js

```diff
@@ -72,4 +76,84 @@ MeetingsUtil.checkForCorrelationId = (deviceUrl, locus) => {
   return false;
 };
 
+
+/**
+ * Will check to see if specified media codec is supported.
+ * @param {String} codec Media codec.
+ * @returns {Boolean}
+ */
+MeetingsUtil.hasMediaCodec = (codec) => {
+  let hasCodec = false;
+  const video = window.document.createElement('video');
+  const PROBABLY = 'probably';
+
+  if (window.MediaSource) {
+    hasCodec = window.MediaSource.isTypeSupported(codec);
```

Comment:
`window.MediaSource.isTypeSupported(codec)` will return `true` or `false` if the codec is supported.